Extension/src/LanguageServer/settings.ts

```diff
@@ -959,6 +960,7 @@ export class OtherSettings {
     public get filesExclude(): vscode.WorkspaceConfiguration | undefined { return vscode.workspace.getConfiguration("files", this.resource).get("exclude"); }
     public get filesAutoSaveAfterDelay(): boolean { return vscode.workspace.getConfiguration("files").get("autoSave") === "afterDelay"; }
     public get InlayHintsEnabled(): boolean { return vscode.workspace.getConfiguration("editor.inlayHints").get<string>("enabled") !== "off"; }
```

Comment:
I think these should be named editorInlayHintsEnabled (pre-existing issue) and editorParameterHintsEnabled to be consistent with the naming of the other settings.

Reply:
Are you talking about these lines? https://github.com/microsoft/vscode-cpptools/blob/8a004672460fc23ea5fa0724148af43c60fab661/Extension/src/LanguageServer/settings.ts#L962-L963
https://github.com/microsoft/vscode-cpptools/blob/8a004672460fc23ea5fa0724148af43c60fab661/Extension/src/LanguageServer/settings.ts#L965

Reply:
Yes